lib/Folder/FolderManager.php

```diff
@@ -146,39 +172,64 @@ public function getAllFoldersWithSize($rootStorageId): array {
 	 * @psalm-return array<int, list<mixed>>
 	 */
 	private function getAllFolderMappings(): array {
-		$query = $this->connection->getQueryBuilder();
-		$query->select('*')
-			->from('group_folders_manage');
+		$queryHelper = $this->circlesManager->getQueryHelper();
+		$query = $queryHelper->getQueryBuilder();
+
+		$query->select('folder_id', 'mapping_type', 'mapping_id')
+			  ->from('group_folders_manage', 'm');
+
+		$queryHelper->addCircleDetails('m', 'mapping_id');
```

Comment:
Depending on if circles is then mandatory or if groups can still be used (which I'd vote for), we should have a possible or move anything circles related to a dedicated services to also allow the app to work if circles is disabled.